sample-input/benchmarks/c5g7/c5g7-3d-rodded-B.py

```diff
@@ -0,0 +1,526 @@
+import openmoc
+
+###############################################################################
```

Comment:
I contemplated adding another geometry file, but it won't reduce the code in the input files since the rodded B 3d geometry input is used in only one place. In the future when we have the full 3D implemented, we will have the three versions of the C5G7 geometry (`c5g7`, `c5g7-rodded-A`, and `c5g7-rodded-B`). How about I add the rodded A geometry and then have 3 geometry files: `geometry.py`, `geometry_rodded_A.py`, and `geometry_rodded_B.py`. The `c5g7.py`, `c5g7-gpu.py`, and `c5g7-cmfd.py` will used `geometry.py` and input files for the rodded A and rodded B cases will use their respective input files. @wbinventor - what do you think of that?